packages/trader/src/App/Components/Elements/VideoStream/playback-rate-control.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Dropdown, Icon } from '@deriv/components';
+
+type TPlaybackRateControl = { onPlaybackRateChange: (new_value: number) => void; is_mobile?: boolean };
+
+const PlaybackRateControl = ({ onPlaybackRateChange, is_mobile }: TPlaybackRateControl) => {
+    const [playback_speed, setPlaybackSpeed] = React.useState('1');
+
+    const playback_rate_list = [
+        { text: '0.25x', value: '0.25' },
+        { text: '0.5x', value: '0.5' },
+        { text: '0.75x', value: '0.75' },
+        { text: 'Normal', value: '1' },
```

Comment:
translate?

Reply:
Yep, didn't notice, that Dropdown has no localize :(